File: PyToCMigration/src/Utility/isTeamAlive.cpp
#include "Characters/BaseCharacter.h"

bool isTeamAlive(std::vector<BaseCharacter> team )
{
    for (int i = 0; i < team.size(); i++)
    {
        if (team[i].alive == true)
        {
            return true;
        }
        
    }
    return false;
}

File: everything_else/ai/finiteStateMachine/combatSimulator/utility/isTeamAlive.py
def isTeamAlive(team):
    '''
    Input: takes in a array of charaters representing a team
    checks isAlive for each one
    output: returns true if at least one member is alive, false otherwise 
    '''
    for char in team:
        if char.isAlive:
            return True

    return False
